src/Features/CSharp/Portable/EmbeddedLanguages/CSharpEmbeddedLanguageCodeFixProvider.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.CodeAnalysis.CSharp.EmbeddedLanguages
     internal class CSharpEmbeddedLanguageCodeFixProvider : AbstractEmbeddedLanguageCodeFixProvider
     {
         public CSharpEmbeddedLanguageCodeFixProvider()
-            : base(CSharpEmbeddedLanguageProvider.Instance)
+            : base(CSharpEmbeddedLanguagesProvider.Instance)
```

Comment:
This was due to the rename to `IEmbeddedLanguageProvider->IEmbeddedLanguagesProvider` done in the "classify normal string escape characters" PR.  This is the downstream fallout of that.